Library/Homebrew/utils/git.rb

```diff
@@ -14,6 +14,19 @@ def last_revision_commit_of_file(repo, file, before_commit: nil)
     out.chomp
   end
 
+  def last_revision_commit_of_files(repo, file, before_commit: nil)
+    args = [before_commit.nil? ? "--skip=1" : before_commit.split("..").first]
+
+    cmd = [
+      HOMEBREW_SHIMS_PATH/"scm/git", "-C", repo,
+      "log", "--format=%h", "--abbrev=7", "--max-count=1", "--name-only",
+      *args, "--", file
+    ]
+    out, = Open3.capture3("/bin/bash", "-c", cmd.join(" "))
```

Comment:
It seems like a very bad idea to be using `bash` here with arguments that are user supplied; it would be very easy to trick users into running commands they didn't want to.

Reply:
I see. I use `bash` there because without it the globstar `**` wouldn't work. I will try another way to search for the exact file path then.

Reply:
Great, thanks ❤️